src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -575,6 +576,22 @@ public bool EnableNotificationCenter
             }
         }
 
+        /// <summary>
+        /// This defines if user wants to the Extensions settings to persist across sessions.
```

Comment:
`This defines if user wants the Extensions settings to persist across sessions.`